benchmarks/decimal_scaling.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const Benchmark = require('benchmark');
+const Decimal = require('decimal.js');
+const { RoundingMode, MC } = require('../lib/bigdecimal.js');
+const { bigDecimals, bigDecimalsBigjs, bigDecimalsBigNumber, bigDecimalsDecimal, bigDecimalsGWT } = require('./test_numbers');
+const { attachEventsAndRun } = require('./utils.js');
+
+const suite = new Benchmark.Suite('Decimal scale');
+
+const scales = [10, 1, 0];
+const length = scales.length;
+
+suite.add('Bigdecimal.js', function () {
+    for (let i = 0; i < bigDecimals.length - 1; i++) {
+        bigDecimals[i].setScale(scales[i%length], RoundingMode.HALF_UP);
```

Comment:
I think you are right, we should remove this suite and the integer rounding to avoid confusing people.

Real usage would likely require to check the decimal places before using setScale to mimic the rounding method provided by other libraries to avoid the extra cost. 

On the other end, if the intent is to update the scale, those libraries doesn't really provide such feature as they don't track the scale like BigDecimal does, and therefore the comparison can't be performed properly.